AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -313,6 +324,11 @@ - (UITextView *)textView
   return _textKitComponents.textView;
 }
 
+- (void)setMaximumLinesToDisplay:(NSUInteger)maximumLines
+{
+  _maximumLinesToDisplay = maximumLines;
```

Comment:
I believe we should add a `[self setNeedsLayout]` call here, to handle the unlikely case that the user changes this property on a live node.